Sources/Kitura/RouterRequest.swift

```diff
@@ -155,6 +155,21 @@ public class RouterRequest {
         parsedURL = URLParser(url: serverRequest.url, isConnect: false)
         url = String(serverRequest.urlString)
         headers = Headers(headers: serverRequest.headers)
+        queryParameters = RouterRequest.decode(queryParameters: parsedURL.queryParameters)
```

Comment:
@shmuelk is this implementation OK ? made `queryParameters` let variable, which is set in `init()`